opal/util/arch.h

```diff
@@ -199,7 +199,8 @@
 
 /* BYTE 2 */
 #define OPAL_ARCH_LONGISxx     0x0000c000 /* mask for sizeof long */
```

Comment:
this one should now be 0x00004000 (aka. OPAL_ARCH_LONGIS32 | OPAL_ARCH_LONGIS64).

Reply:
Yeah, since it's only giving two options vs 3 for a lot of the other "*ISxx" masks I'm fine with changing that to only have the one bit.